CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing
+
+## Contributor's Agreement
+
+You are under no obligation whatsoever to provide any bug fixes,
+patches, or upgrades to the features, functionality or performance of
+the source code ("Enhancements") to anyone; however, if you choose to
+make your Enhancements available either publicly, or directly to the
+project, without imposing a separate written license agreement for
+such Enhancements, then you hereby grant the following license: a
+non-exclusive, royalty-free perpetual license to install, use, modify,
+prepare derivative works, incorporate into other computer software,
+distribute, and sublicense such enhancements or derivative works
+thereof, in binary and source code form.
+
+## [Technical Charter](https://apptainer.org/technical-charter)
+
+- section 2(b) requires a list of Technical Steering Committee (TSC)
+  members here
+
+### Technical Steering Committee
+
+(in alphabetical sort order of surname)
+
+```text
+- Jonathon Anderson <jonathon.anderson@ciq.co>, <anderbubble@gmail.com>
+- Christian Goll <christian.goll@googlemail.com>
+- John Hanks <griznog@gmail.com>
+- Gregory Kurtzer <gmkurtzer@gmail.com>
+- Jeremy Siadal <jeremy.c.siadal@intel.com>
+```
+
+## Getting Started
+
+When contributing to Warewulf, it is important to properly communicate
+the gist of the contribution. If it is a simple code or editorial fix,
+simply explaining this within the GitHub Pull Request (PR) will
+suffice. But if this is a larger fix or Enhancement, you are advised
+to first discuss the change with the project leader or developers.
+
+<!--
+Please note we have a [code of conduct](CODE_OF_CONDUCT.md). Please
+follow it in
+all your interactions with the project members and users.  
+
+In addition, all commits must be "Signed-off" by using `git commit
+-s`,
+acknowledging that you agree to the [Developer Certificate of Origin](DCO.md).  
+-->
+
+## Pull Requests (PRs)
+
+1. Essential bug fix PRs should be sent to both main and release
```

Comment:
Instead of saying essential, I would be specific.

"All bug fixes and regressions should be submitted against the development and main branch."
"All security fixes need to be submitted against the development branch, main branch and currently released tag"

Reply:
I agree with making it more specific; but I think more discussion needs to be had about the branching strategy in order for that specificity to be correct. I'm afraid that the way people are currently talking about development vs main will lead to painful merges or (more like what's happening now) a divergence that makes one of them not as well used or tested. But rather than try to figure that out (or make my arguments) at this stage, I figured the lesser specificity (using the language from Apptainer) would make it easier to get something down, and then we could discuss the specifics next.